packages/babel-generator/src/generators/modules.js

```diff
@@ -45,12 +45,6 @@ export function ExportAllDeclaration(node: Object) {
   this.word("export");
   this.space();
   this.token("*");
-  if (node.exported) {
```

Comment:
I was thinking it is ExportNamedDeclaration -- https://github.com/babel/babylon/pull/367